src/start.sh

```diff
@@ -49,34 +46,16 @@ start() {
     fi
   fi
 
-  fix_capabilities
-  # validate_env || exit 1
-  ensure_basic_configuration
-
-  apply_FTL_Configs_From_Env
-
-  # Web interface setup
-  # ===========================
-  # load_web_password_secret
-  # setup_web_password
-
-  # Misc Setup
-  # ===========================
-  # setup_blocklists
-
-  # FTL setup
-  # ===========================
-
-  # setup_FTL_User
-  # setup_FTL_query_logging
-
-  [ -f /.piholeFirstBoot ] && rm /.piholeFirstBoot
+  # If FTLCONF_dns_upstreams contains any semicolons, replace them with commas
```

Comment:
Do we really need this? We could fail hard here, make the user change from semicolons to commas and remove the hand-holding in a few versions.

Reply:
I guess no, not strictly - but I figured it was a more friendly way of just breaking the containers that are already out there.

Reply:
FTL can accept `;` as well, we just make

https://github.com/pi-hole/FTL/blob/4eca25e2a5700a35523fc62560447082964d00ae/src/config/toml_helper.c#L936

``` c
const char delim[] =",;";
```

Reply:
However you prefer

Reply:
Less code over there, fewer migration things to handle correctly by just supporting this: https://github.com/pi-hole/FTL/pull/1690

Reply:
I will try this once it builds and force push away the change from this side